src/components/Form/FormFields/TagInput.vue

```diff
@@ -189,7 +192,7 @@ export default {
 }
 
 .filter-field ::v-deep .el-input__inner {
-  height: 28px !important;
+  height: 27px !important;
 }
 
 .show-password {
```

Comment:
I'm sorry, but I cannot see the text you provided because it seems to be cut off due to a knowledge cutoff of 2021-09-01. Could you please share that portion with me?